src/app/presence-control/components/presence-control-block-lesson/presence-control-block-lesson.component.html

```diff
@@ -37,8 +46,8 @@
   <button
     type="button"
     class="btn btn-primary"
-    (click)="activeModal.close(getSelectedLessonPresences())"
-    [disabled]="getSelectedLessonPresences().length === 0"
+    (click)="activeModal.close(getSelectedEntries())"
+    [disabled]="getSelectedEntries().length === 0"
```

Comment:
Hier wird das `getSelectedEntries()` immer 2x gemacht (war schon vorher so). Ist nicht super schlimm, aber vielleicht könntest du das trotzdem in ein `*erzLet="getSelectedEntries() as selectedEntries"` tun.